src/main/java/org/systemexception/lunarlander/main/TestJbox.java

```diff
@@ -21,62 +29,149 @@
+			Display.sync(60);
 		}
+		Display.destroy();
+		System.exit(0);
 	}
 
-	private static void logic() {
-		world.step(1 / 60f, 8, 3);
+	private void initGL() {
+		try {
+			Display.setDisplayMode(new DisplayMode(WINDOW_DIMENSIONS[0], WINDOW_DIMENSIONS[1]));
+			Display.setTitle(WINDOW_TITLE);
+			Display.setVSyncEnabled(true);
+			Display.create();
+		} catch (LWJGLException e) {
+			e.printStackTrace();
+			System.exit(0);
+		}
+
+		glMatrixMode(GL_PROJECTION);
+
+		glShadeModel(GL11.GL_SMOOTH);
+		glDisable(GL11.GL_TEXTURE_2D);
+		glDisable(GL11.GL_DEPTH_TEST);
+		glDisable(GL11.GL_LIGHTING);
+
+		glClearColor(0.0f, 0.0f, 0.0f, 0.0f);
+		glClearDepth(1);
+
+		glEnable(GL11.GL_BLEND);
+		glBlendFunc(GL11.GL_SRC_ALPHA, GL11.GL_ONE_MINUS_SRC_ALPHA);
+
+		glViewport(0, 0, Display.getWidth(), Display.getHeight());
+		glMatrixMode(GL11.GL_MODELVIEW);
+
+		glMatrixMode(GL11.GL_PROJECTION);
+		glLoadIdentity();
+		glOrtho(0, Display.getWidth(), Display.getHeight(), 0, 1, -1);
+		glMatrixMode(GL11.GL_MODELVIEW);
 	}
 
-	private static void input() {
-		for (Body body : bodies) {
-			if (body.getType() == BodyType.DYNAMIC) {
-				if (Keyboard.isKeyDown(Keyboard.KEY_A)) {
-					body.applyAngularImpulse(+0.01f);
-				} else if (Keyboard.isKeyDown(Keyboard.KEY_D)) {
-					body.applyAngularImpulse(-0.01f);
-				}
-				if (Keyboard.isKeyDown(Keyboard.KEY_W)) {
-					body.applyLinearImpulse(new Vec2(0, 0.05f), body.getPosition());
-				}
-				if (Mouse.isButtonDown(0)) {
-					Vec2 mousePosition = new Vec2(Mouse.getX(), Mouse.getY()).mul(0.5f).mul(1 / 30f);
-					Vec2 bodyPosition = body.getPosition();
-					Vec2 force = mousePosition.sub(bodyPosition);
-					body.applyForce(force, body.getPosition());
-				}
-			}
+	public void initFonts() {
+		try {
+			InputStream inputStream = ResourceLoader.getResourceAsStream("ubuntu.ttf");
+			Font awtFont = Font.createFont(Font.TRUETYPE_FONT, inputStream);
+			awtFont = awtFont.deriveFont(24f); // set font size
+			font = new TrueTypeFont(awtFont, true);
+
+		} catch (Exception e) {
+			e.printStackTrace();
 		}
 	}
 
-	private static void cleanUp(boolean asCrash) {
-		Display.destroy();
-		System.exit(asCrash ? 1 : 0);
+	private void render() {
+		// Draw box
+		Body box = bodies.get(boxBody);
+		Color.red.bind();
+		glPushMatrix();
+		Vec2 bodyPosition = box.getPosition().mul(30);
+		glTranslatef(bodyPosition.x, bodyPosition.y, 0);
+		glRotated(Math.toDegrees(box.getAngle()), 0, 0, 1);
+		glRectf(-0.75f * 30, -0.75f * 30, 0.75f * 30, 0.75f * 30);
+		glPopMatrix();
+		font.drawString(0, 0, "Position: " + box.getPosition(), org.newdawn.slick.Color.yellow);
+		TextureImpl.bindNone();
+		// Draw box head
+		Body boxHead = bodies.get(boxHeadBody);
+		Color.yellow.brighter().bind();
+		glPushMatrix();
+		glPushMatrix();
+		double v = normalRelativeAngle(box.getAngle());
+		font.drawString(0, 40, "Angle: " + String.format("%.2f", v) + ", Radians: " + box.getAngle(),
+				org.newdawn.slick.Color.yellow);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.newdawn.slick.Color.yellow' due to existing import 'org.newdawn.slick.Color'](https://www.codacy.com/app/leo_4/lunar-lander/file/1690538463/issues/source?bid=3040650&fileBranchId=3043029#l130)